packages/x-data-grid/src/components/containers/GridRootStyles.ts

```diff
@@ -535,6 +535,7 @@ export const GridRootStyles = styled('div', {
 
     /* Cell styles */
     [`& .${c.cell}`]: {
+      flex: '0 0 auto',
```

Comment:
The list column shrinks without this - it was taking up the right amount of space before due to the minWidth, maxWidth. These were removed recently https://github.com/KenanYusuf/mui-x/commit/315ffa1e163a0bdd54a89909cf1dd7ffa582cd3a

Reply:
Any opinion if we should add back min/max instead of flex auto?

Reply:
My preference is setting the `flex`. We only have to set this property once, and the alternative requires updating `min-width` and `max-width` in [various places](https://github.com/mui/mui-x/pull/14448/files) to be in sync with the column width.